### PR TITLE
Content updates

### PR DESCRIPTION
**Pull request by leighmoney**

### CODE REVIEW

File: source/public/mvp-1/confirmation.slim [hunk 1]
```diff
@@ -17,9 +17,8 @@ form
       hr
       br
       h4 What happens next?
-      p We've sent your application to the court.
-      p They will contact you either to confirm your application, or to ask for more information.
-      p If you have any questions about your application, contact <a href"#"> the call centre</a>.
+      p Your application will be processed. We'll let you know if your application has been unsucessful or if we need more information from you.
```
Comment (ghost) [inline source/public/mvp-1/confirmation.slim:20]: Typo 'unsuccessful'

File: source/public/mvp-1/probate.slim [hunk 2]
```diff
@@ -8,7 +8,7 @@ form
     .small-12.columns
       br
       h4 Are you paying a fee for a probate case?
-      p These cases are usually about the property and belongings of someone who has died.
+      p Probate cases are about the property and belongings of someone who has died.
```
Comment (ghost) [inline source/public/mvp-1/probate.slim:11]: It says 'These cases' on the paper form, but no problem if you want it to be different online

File: source/public/mvp-1/status.slim [hunk 3]
```diff
@@ -7,18 +7,18 @@ form
   .row
     .small-12.medium-6.columns    
       br
-      h4 What is your status?
-      p If your case is against your partner, such as a divorce, dissolution or domestic violence, select single.
+      h4 Are you married or living with someone and sharing an income?
```
Comment (ghost) [inline source/public/mvp-1/status.slim:10]: The paper form asks 'What is your status?', no problem if it is different online, but I thought we found these 'yes/no' questions cause too much cognitive load.

File: source/public/mvp-1/probate.slim (hunk 2 above)
Comment (leighmoney) [inline source/public/mvp-1/probate.slim:11]: Yes, I prefer it without 'these cases' and the 'usually'

**Pull request merged**